src/app/_services/api.service.ts

```diff
@@ -10,58 +10,71 @@ const baseUrl = environment.apiUrl;
 @Injectable()
 export class ApiService {
 
-  constructor(private http: HttpClient) { }
+    constructor(private http: HttpClient) { }
 
-  getHttpHeaders() {
-  	let rtn = {
-  		headers: new HttpHeaders({
-  			'Content-Type': 'application/json'
-  		})
-  	}
+    getHttpHeaders() {
+          let rtn = {
+                  headers: new HttpHeaders({
+                          'Content-Type': 'application/json'
+                  })
+          }
 
-  	return rtn;
-  }
+          return rtn;
+    }
 
-  // http.get needs an Authorization in header
-  getHttpHeadersWithBase64Authorization(username:string, password: string) {
-  	let rtn = {
-  		headers: new HttpHeaders({
-        'Authorization': "Basic " + btoa(`${username}:${password}`),
-  			'Content-Type': 'application/json'
-  		})//,
-      //responseType: 'json'
-  	}
+    // http.get needs an Authorization in header
+    getHttpHeadersWithBase64Authorization(username:string, password: string) {
+          let rtn = {
+                  headers: new HttpHeaders({
+          'Authorization': "Basic " + btoa(`${username}:${password}`),
+                          'Content-Type': 'application/json'
+                  })//,
+        //responseType: 'json'
+          }
 
-  	return rtn;
-  }
+          return rtn;
+    }
 
-  get(url:string, username:string, password:string) {
-    return this.http.get<any>(baseUrl + url,
-      this.getHttpHeadersWithBase64Authorization(username, password))
-      .pipe(
-        catchError(this.handleError)
-      );
-  }
+    get(url:string, username:string, password:string) {
+      return this.http.get<any>(baseUrl + url,
+        this.getHttpHeadersWithBase64Authorization(username, password))
+        .pipe(
+          catchError(this.handleError)
+        );
+    }
 
-  post(url, data) {
-  	return this.http.post(baseUrl + url, data, this.getHttpHeaders())
-  }
+/* 
```

Comment:
You usually should not commit commented out code. If you just want to remember what it was like before you changed it, that is what source control (git) is for.

Reply:
Removed.